meta-xt-driver-domain/recipes-extended/qemu-command-handler/qemu-command-handler.bb

```diff
@@ -0,0 +1,37 @@
+SUMMARY = "Service for the block device up notification"
+
+PV = "0.1"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+SRC_URI = " \
+    file://qemu-command-handler.service \
+    file://qemu-command-handler-ExecStart.sh \
+"
+S = "${WORKDIR}"
+
+inherit systemd
+
+SYSTEMD_SERVICE:${PN} = "qemu-command-handler.service"
+
+RDEPENDS:${PN} = " \
+    xen-tools-xenstore \
+    bash \
+"
+
+RRECOMMENDS:${PN} += " \
+    virtual/xenstored \
```

Comment:
Hi @lorc, I already do not remember, why that was added. I'll try without it. If it will work - I'll remove it. If not - I'll come back with additional status.

Reply:
Hi @lorc, done. Only xen-tools-xenstore has left.